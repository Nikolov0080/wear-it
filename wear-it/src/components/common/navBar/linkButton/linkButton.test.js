import React from 'react';
import renderer from 'react-test-renderer';
import LinkButton from './linkButton';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Link Button', () => {
    const tree = renderer
        .create(<BrowserRouter><LinkButton /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});