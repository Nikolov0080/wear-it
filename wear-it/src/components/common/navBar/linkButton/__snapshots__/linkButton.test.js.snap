// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Loading 1`] = `
<a
  className="linkBtn"
  href="/da"
  onClick={[Function]}
>
  <div
    className="btn"
  >
    dssd
  </div>
</a>
`;
